packages/runner/lib/workers/worker.integration.test.ts

```diff
@@ -0,0 +1,140 @@
+import { expect, describe, it, vi, beforeAll, afterAll } from 'vitest';
+import type { DBSyncConfig, NangoProps } from '@nangohq/types';
+import { RunnerWorker } from './worker.js';
+import express from 'express';
+import type { Server } from 'node:http';
+import getPort from 'get-port';
+import { Locks } from '../sdk/locks.js';
```

Comment:
Not sure what is your setup but it's not auto-fixing the warning for the import ordering